src/Config/Loader/PhpFileLoader.php

```diff
@@ -52,7 +52,29 @@ public function load($file, $type = null)
             $code = substr($code, 0, -2);
         }
 
-        return rtrim($code)."\n";
+        // declare() statements
+        $lines = explode("\n", $code);
+        $codeNew = '';
+        $processed = false;
+
+        foreach ($lines as $line) {
+            if ('' === $line || $processed) {
```

Comment:
Ah true, was just a regular thought 😄  Want to PR your changes against my PR? It's your code after all 😄